web/src/pages/projectInsights/DetailsTable.tsx

```diff
@@ -0,0 +1,903 @@
+// DetailsTable.tsx
+import React, { useState } from 'react'
+import { ProjectInsightsDetails } from '../../sm-api'
+import { Button, Table, Icon } from 'semantic-ui-react'
```

Comment:
Instead of the table here, look for the other defined Table in this function, you'll still want to import Table as SUITable so you can access the SUITable.Row / .Cell, etc. But our table definition has some handy dark mode, compact customisations